### DISCUSSION TITLE
Welcome to #tofuengine Discussions!

### DISCUSSION THREAD

**Discussion by MarcoLizza** [tofuengine/tofu#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheKing0x9** [comment]
Hi,
I'm Aayush Kashyap, and I've been programming games for the past few years. Recently, I decided to move outside of the environment that I had been comfortable with for a long while and explore new frameworks and engines for game development. Luckily I found Tofu Engine some time back and it really caught my eye. I like how it has been structured as a whole, and since it is written in C and Lua, I hope I can be more useful and contribute to it too :wink:
